services/venice-router/src/main/java/com/linkedin/venice/router/MetaDataHandler.java

```diff
@@ -552,12 +553,44 @@ private void handleBlobDiscovery(ChannelHandlerContext ctx, VenicePathParserHelp
           Integer.parseInt(storeVersion),
           Integer.parseInt(storePartition),
           Optional.empty());
+
+      if (!instances.isEmpty()) {
+        LOGGER.info(
```

Comment:
This seems to be redundant logging from your debugging session. We can log when no instances are found, but when there are instances found, for production use cases, there might be thousands of replicas for certain Da Vinci use cases. Let's not log it there, or at least, just log the number of the instances.

Also, can you only log these once? Try to merge with below logging line. 
Also, nit: Please update the name to liveNodeHostNames to something like readyToServeNodeHostNames, and update the corresponding log message, as we are for now not checking whether it is alive instance, but rather checking its push readiness.

Reply:
please double check the logging changes